modules/hcp-updating-requirements.adoc

```diff
@@ -6,16 +6,16 @@
 [id="hosted-control-planes-upgrading-requirements_{context}"]
```

Comment:
I was unable to find this file in the preview. According to its metadata it should be in this assembly: https://76266--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-updating . However it is not there. Please check what's going on here.

Reply:
Rebased this PR and it shows the file fine now.